core/src/main/java/greencity/converters/UserArgumentResolver.java

```diff
@@ -26,7 +29,7 @@
     @Override
     public boolean supportsParameter(MethodParameter parameter) {
         return parameter.getParameterAnnotation(CurrentUser.class) != null
-            && parameter.getParameterType().equals(User.class);
+            && parameter.getParameterType().equals(User.class) || parameter.getParameterType().equals(UserVO.class);
```

Comment:
Not sure, but looks like you need () here -> (parameter.getParameterType().equals(User.class) || parameter.getParameterType().equals(UserVO.class)). Or I am wrong ?

Reply:
Changes done.